src/analysis/souffle/operations.dl

```diff
@@ -41,29 +41,29 @@ isOperation([owner, operator, result, [input1, [input2, nil]], attrs])
 
 // Explode the operand lists so that we can extract the heads of each sublist
 // to get a mapping from operations to their operands.
-.decl operationToPartialOperandLists(operation: Operation, partialList: OperandList)
+.decl operationHasOperandHelper(operation: Operation, partialList: OperandList)
 
-operationToPartialOperandLists([owner, op, result, operandList, attrs], operandList) :-
+operationHasOperandHelper([owner, op, result, operandList, attrs], operandList) :-
   isOperation([owner, op, result, operandList, attrs]).
-operationToPartialOperandLists(operation, tail) :-
-  operationToPartialOperandLists(operation, [_, tail]).
+operationHasOperandHelper(operation, tail) :-
+  operationHasOperandHelper(operation, [_, tail]).
```

Comment:
Not wild about renaming this to just "helper": the partial suffixes currently reflected in the relation name allow this relation to be sensibly reused for multiple purposes without impacting readability. With the renaming, it is closely tied to this one purpose only.

We aren't using it for anything else right now, but I am going to reuse this partial state to calculate the index at which various operands reside in a future PR.

Reply:
Changed. Made it `operandHasOperandListSuffix`. PTAL.